ha/resource/sspl

```diff
@@ -152,7 +152,7 @@ stateful_demote() {
 }
 
 stateful_promote() {
-    sleep 5
+    sleep 8
```

Comment:
I think we can restore this sleep back to 5 seconds and even less than 5 seconds, since the salt-calls in sspl code have been converted to using provisioner APIs

cc @saurabhkhanvilkar @Madhura-08

Reply:
@indrajitzagade i think we can take this as separate pr as it is only to port to main.
i am creating issue for this.

Reply:
https://github.com/Seagate/cortx-ha/issues/206

Reply:
@indrajitzagade i remember @Madhura-08 had tried that and going below 5 was still not working even after her changes... i think let's keep it 8 for now.

Reply:
@saurabhkhanvilkar  i have created separate issue for this we can check it separately

This pr only to port from cortx to main

we can disscuss more on sleep time
Issue: https://github.com/Seagate/cortx-ha/issues/206